modules/ROOT/pages/slsa.adoc

```diff
@@ -0,0 +1,156 @@
+
+= Red Hat Trusted Application Pipeline, Enterprise Contract, and SLSA
+
+.Overview
+****
+
+Red Hat Trusted Application Pipeline is a build and CI/CD system
+that aims to meet or exceed https://slsa.dev/spec/v1.0/levels[SLSA Build L3 (v1.0)].
+Enterprise Contract is an
+https://slsa.dev/spec/v1.0/verifying-artifacts[artifact verifier] and
+customizable policy checker designed to be easily integrated with RHTAP's CI/CD
+workflows.
+
+A lot of the default policies created for RHTAP and enforced by EC are closely
+related to SLSA requirements, based on both the recent SLSA v1.0, and the older
+SLSA v0.1 specification. In this documentation we'll describe in more detail
+how RHTAP and EC work together to provide a high standard of supply chain
+security, informed and guided by the link:https://slsa.dev/[SLSA
+specifications].
+
+****
+
+== SLSA Requirements
+
+=== SLSA requirements verified explicitly by Enterprise Contract
+
+Enterprise Contract comes with a set of policy rules designed for use with
+RHTAP. Using EC to verify that a build passes all the checks defined by the
+policy provides a high level of confidence that the build was built securely
+and can be trusted.
+
+Many of the policy rules are linked to specific SLSA requirements, for example:
+
+[cols="1,1,3"]
+|===
+|SLSA reference|EC policy rule|Notes
+
+a| https://slsa.dev/spec/v0.1/requirements#scripted-build[Scripted build (v0.1)]
+a| xref:ec-policies:ROOT:release_policy.adoc#slsa_build_scripted_build__build_script_used[Build task contains steps]
+a| RHTAP uses link:https://tekton.dev/[Tekton] so knowing that is probably enough to consider this requirement satisfied, however this specific
```

Comment:
```suggestion
a| RHTAP uses link:https://tekton.dev/[Tekton] so knowing that is enough to consider this requirement satisfied, however this specific
```

Reply:
Just, stronger language.

It will leave a better impression since this is the _first_ table row that the reader encounters.